Redeclared Template Errors

Updates to Trilinos have caused such errors as:
```
In file included from ...NaluNightlyTesting/Nalu/include/LinearSystem.h:12:0,
                 from ...NaluNightlyTesting/Nalu/src/AssembleContinuityEdgeSolverAlgorithm.C:14:
...NaluNightlyTesting/Nalu/include/LinearSolverTypes.h:38:7: error: redeclared with 5 template parameters
 class MultiVector;
       ^
In file included from ...NaluNightlyTesting/Nalu/include/LinearSystem.h:12:0,
                 from ...NaluNightlyTesting/Nalu/src/AssembleMomentumEdgeSolverAlgorithm.C:13:
...NaluNightlyTesting/Nalu/include/LinearSolverTypes.h:38:7: error: redeclared with 5 template parameters
 class MultiVector;
       ^
In file included from ...NaluNightlyTesting/spack/opt/spack/linux-centos7-x86_64/gcc-4.9.2/trilinos-develop-od2ztan4e3j3nwawssodh474lo5jvbhk/include/Tpetra_Vector_decl.hpp:54:0,
                 from ...NaluNightlyTesting/spack/opt/spack/linux-centos7-x86_64/gcc-4.9.2/trilinos-develop-od2ztan4e3j3nwawssodh474lo5jvbhk/include/Tpetra_RowMatrix_decl.hpp:49,
                 from ...NaluNightlyTesting/spack/opt/spack/linux-centos7-x86_64/gcc-4.9.2/trilinos-develop-od2ztan4e3j3nwawssodh474lo5jvbhk/include/Tpetra_CrsMatrix_decl.hpp:54,
                 from ...NaluNightlyTesting/spack/opt/spack/linux-centos7-x86_64/gcc-4.9.2/trilinos-develop-od2ztan4e3j3nwawssodh474lo5jvbhk/include/Tpetra_CrsMatrix.hpp:1,
                 from ...NaluNightlyTesting/Nalu/include/LinearSolverTypes.h:14,
                 from ...NaluNightlyTesting/Nalu/include/LinearSystem.h:12,
                 from ...NaluNightlyTesting/Nalu/src/AssembleMomentumEdgeSolverAlgorithm.C:13:
...NaluNightlyTesting/spack/opt/spack/linux-centos7-x86_64/gcc-4.9.2/trilinos-develop-od2ztan4e3j3nwawssodh474lo5jvbhk/include/Tpetra_MultiVector_decl.hpp:75:56: note: previous declaration 'template<class S, class LO, class GO, class N> class Tpetra::MultiVector' used 4 template parameters
   template<class S, class LO, class GO, class N> class MultiVector;
```

--- a/include/LinearSolverTypes.h
+++ b/include/LinearSolverTypes.h
@@ -12,6 +12,8 @@
 #include <KokkosInterface.h>
 #include <Tpetra_CrsGraph.hpp>
 #include <Tpetra_CrsMatrix.hpp>
+#include <Tpetra_Vector.hpp>
+#include <Tpetra_MultiVector.hpp>
 
 // Forward declare templates
 namespace Teuchos {
@@ -33,12 +35,6 @@ class Map;
 
 template <typename LocalOrdinal, typename GlobalOrdinal, typename Node >
 class Export;
-
-template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Node, bool classic>
-class MultiVector;
-
-template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Node, bool classic>
-class Vector;
 
 template <typename Scalar, typename LocalOrdinal, typename GlobalOrdinal, typename Node>
 class Operator;

